### PR TITLE
Master property field small icon

### PR DESCRIPTION
**Pull request by msh-odoo**
Task:https://www.odoo.com/web#id=12208&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Pad: https://pad.odoo.com/p/r.3yb55EByLFEH5nBH

### CODE REVIEW

File: addons/web/static/src/js/views/form_common.js [hunk 1]
```diff
@@ -773,6 +776,21 @@ var AbstractField = FormWidget.extend(FieldInterface, {
             self.do_action(r);
         });
     },
+    on_property_open: function(e) {
+        var self = this;
+        e.stopImmediatePropagation();
+        new data.DataSetSearch(self, 'ir.model.fields').call('search',
```
Comment (rga-odoo) [inline addons/web/static/src/js/views/form_common.js]: can we create server side method which return action ?

File: addons/web/static/src/js/views/form_widgets.js [hunk 2]
```diff
@@ -622,6 +624,8 @@ var FieldBoolean = common.AbstractField.extend({
         }, this));
         var check_readonly = function() {
             self.$checkbox.prop('disabled', self.get("effective_readonly"));
+            //Note: FieldBoolean does not have ReinitializeFieldMixin, so we need to toggle visibility class based on effective_readonly
+            self.$el.find(".oe_field_property").toggleClass('o_hidden', !!self.get("effective_readonly"));
```
Comment (rga-odoo) [inline addons/web/static/src/js/views/form_widgets.js]: `self.$('.oe_field_property')`

File: addons/web/static/src/xml/base.xml [hunk 3]
```diff
@@ -905,6 +909,7 @@
             t-att-tabindex="widget.node.attrs.tabindex"
             t-att-autofocus="widget.node.attrs.autofocus"
             class="field_boolean"/>
+            <i t-if="widget.field.company_dependent and widget.session.debug" class="oe_field_property oe_field_property_no_margin o_hidden fa fa-cog"/>
```
Comment (rga-odoo) [inline addons/web/static/src/xml/base.xml]: create common template for property fields

File: addons/web/static/src/js/views/form_common.js (hunk 1 above)
Comment (msh-odoo) [inline addons/web/static/src/js/views/form_common.js]: :+1:

**Comment by hmo-odoo** [comment]
continue work on this PR: https://github.com/odoo/odoo/pull/12102

**Pull request closed without merging**